README.md

```
@@ -1,269 +1,629 @@
+
+### `gitb reset <mode>`
+
+| **Mode**      | **Short** | **Description** | **Use Case** |
+|---------------|-----------|-------------------------------------------------------------------------|--------------|
+| `<empty>`     |           | Reset last commit (mixed mode) | Undo last commit, keep changes |
+| `soft`        | `s`       | Soft reset last commit | Redo commit message |
+| `undo`        | `u`       | Undo last reset operation | Recover from mistake |
+| `interactive` | `i`       | Select commit to reset to | Go back multiple commits |
+| `ref`         | `r`       | Reset to selected HEAD reference | Use reflog recovery |
+
+### `gitb stash <mode>`
```

markdownlint-cli2 (0.17.2)

Multiple headings with the same content
null

(MD024, no-duplicate-heading)